languages/Language.php

```diff
@@ -687,6 +687,32 @@ public static function getLanguageNames( $customisedOnly = false ) {
 	}
 
 	/**
+	 * Return languages supported by fandom
+	 *
+	 * @return array
+	 */
+	public static function getSupportedLanguages() {
+		return [ 'de', 'en', 'es', 'fr', 'it', 'ja', 'pl', 'pt-br', 'ru', 'zh', 'zh-tw' ];
+	}
+
+	/*
+	 * get a list of language names available for wiki request
```

Comment:
`available for wiki request`? Shouldn't this say `available for Create New WIki`?

_Request_ word comes from The Old Time when user had to request a wiki and the creation process was manually run by the community.